public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/appInsightsGeoMap.json

```diff
@@ -0,0 +1,997 @@
+{
+  "__inputs": [],
+  "__elements": [],
+  "__requires": [
+    {
+      "type": "panel",
+      "id": "gauge",
+      "name": "Gauge",
+      "version": ""
+    },
+    {
+      "type": "panel",
+      "id": "geomap",
+      "name": "Geomap",
+      "version": ""
+    },
+    {
+      "type": "grafana",
+      "id": "grafana",
+      "name": "Grafana",
+      "version": "8.5.1"
+    },
+    {
+      "type": "datasource",
+      "id": "grafana-azure-monitor-datasource",
+      "name": "Azure Monitor",
+      "version": "1.0.0"
+    },
+    {
+      "type": "panel",
+      "id": "stat",
+      "name": "Stat",
+      "version": ""
+    },
+    {
+      "type": "panel",
+      "id": "timeseries",
+      "name": "Time series",
+      "version": ""
+    }
+  ],
+  "editable": true,
+  "id": null,
+  "iteration": null,
+  "liveNow": false,
+  "panels": [
+    {
+      "gridPos": {
+        "h": 4,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
```

Comment:
I would make this box a bit bigger so the whole text can fit verticallyif the first line does not fit completely in one line

![Screenshot from 2022-07-25 16-04-58](https://user-images.githubusercontent.com/4025665/180797088-7917896f-3991-43ed-86a9-34532f91be98.png)

Reply:
Also, is there any link to a tutorial or something to let people know how they can configure tests?

Reply:
Thank you for the feedback! I have resized the panel: 
![image](https://user-images.githubusercontent.com/20448042/180922296-b78a294d-f795-425b-8a4e-006982c9594d.png)


As for the link, we have quite a few OOB dashboards where we don't provide links for tutorials on set up. So adding it here would break consistency. I have talked to our PM and decided that for now it is best to not take on the task of adding instructions as the usual discoverability path tends to be Azure Service -> Grafana dashboard. So we can expect most users coming to this dashboard to already know or have availability tests set up.

I have also shared credentials! let me know if you run into any issues.